ush/templates/FV3LAM_wflow.xml

```diff
@@ -963,7 +1126,7 @@ MODULES_RUN_TASK_FP script.
       {%- elif do_dacycle %}
       <taskdep task="&ANAL_GSI_TN;_#type#"/>
       {%- else %}
-      <taskdep task="&PREP_CYC_TN;_#type#{{ uscore_ensmem_name }}"/>
+      <taskdep task="&PREP_CYC_PROD_TN;{{ uscore_ensmem_name }}"/>
```

Comment:
Line 1129 should take the left side form, i.e:
`   <taskdep task="&PREP_CYC_TN;_#type#{{ uscore_ensmem_name }}"/>
`